cogs/bot.py

```diff
@@ -451,14 +451,14 @@ async def pick(self, ctx, *, name: str):
         member = await self.bot.mongo.fetch_member_info(ctx.author)
 
         if member is not None:
-            return await ctx.send(
+            return await ctx.reply(
```

Comment:
Could you do `mention_author=False` for this because we don't want users' mentions tabs to get clogged.